pkg/analysis_server/lib/src/services/correction/fix_internal.dart

```diff
@@ -1784,6 +1787,18 @@ class FixProcessor {
     _addFix(DartFixKind.ADD_NE_NULL, []);
   }
 
+  void _addFix_removeAwait() {
+    AwaitExpression awaitExpression = node is AwaitExpression
+        ? node
+        : node.getAncestor((node) => node is AwaitExpression);
```

Comment:
Will the `node` every *not* be an await expression? Just wondering whether this defensive code could be removed. (Here and in other fix methods.)